Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -367,6 +379,21 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         if (this.delegate && typeof this.delegate.didAppendConsoleMessageView === "function")
             this.delegate.didAppendConsoleMessageView(messageView);
     }
+
+    _handleShowConsoleMessageTimestampsSettingChanged()
+    {
+        this._showTimestamps = WI.settings.showConsoleMessageTimestamps.value;
+        if (this._showTimestamps) {
+            this._currentSessionOrGroup.element.removeAttribute("hide-timestamps");
+            for (let i = 0; i < this._consoleMessageViews.length; i++) {
+                let consoleMessageView = this._consoleMessageViews[i];
+                if (consoleMessageView.constructor.name == "ConsoleMessageView")
+                    consoleMessageView.renderTimestamp();
+            }
```

Comment:
When we don't need the index as part of the iteration it is preferable to use a for-of loop
```
for (let consoleMessageView of this._consoleMessageViews) {
    ...
}
```